tests/code/test_event_list.py

```diff
@@ -0,0 +1,94 @@
+import pytest
+from peewee import DoesNotExist
+from app.models import mainDB
+from app.models.programEvent import ProgramEvent
+from app.models.event import Event
+from app.logic.events import getStudentLeadProgram,  getTrainingProgram, getBonnerProgram, getOneTimeEvents
+
+@pytest.mark.integration
+def test_event_list():
+    with mainDB.atomic() as transaction:
+        studentLead = Event.create(name = "Test Student Lead",
+                                term = 3,
+                                description = "event for testing",
+                                timeStart = "18:00:00",
+                                timeEnd = "21:00:00",
+                                location = "basement",
+                                isRecurring = 0,
+                                isRsvpRequired = 0,
+                                isTraining = 0,
+                                isService = 0,
+                                startDate = 2021-12-12,
+                                endDate = 2021-12-13)
+        studentProgramEvent = ProgramEvent.create(program = 2, event = studentLead)

```

Comment:
if you want to save some space, you don't need to enter values for fields with defaults (like all the boolean ones that are 0)